rackerlabs/cookbook-repose
# 3.3.4 branch

# What
Minor updates to the cookbook - both bug fixes and updates to allow some attributes to be optional (as documented in repose docs).

# Why
Updates to recipes, templates and attributes to allow for us in Rackspace Metrics as a wrapper cookbook model.

# How to test
Check that ip-user template no longer causes an ERROR warning due to missing quotes.  Check that keystone_v2 attributes are only populated if a value is set for the optional attributes.  For the slf4j logging config, test that a CDATA block can be added as a format value in the attributes file.
